Makefile

```diff
@@ -149,11 +150,24 @@ PULSEAUDIO_VARS_OFF=	GN_ARGS+=use_pulseaudio=false
 TEST_DISTFILES=		${CHROMIUM_TEST_FONTS_HASH}:chromium_testfonts
 TEST_ALL_TARGET=	${TEST_TARGETS}
 
-post-fetch:
-	${RM} -r ${TMPDIR}/npm-cache
-	${MKDIR} ${TMPDIR}/npm-cache
-	${CP} ${FILESDIR}/package.json ${FILESDIR}/package-lock.json ${TMPDIR}/npm-cache
-	cd ${TMPDIR}/npm-cache && npm install --verbose --no-progress
+NPM_TIMESTAMP=	1557238990
+
+pre-fetch:
+	if [ ! -f ${DISTDIR}/electron-npm-modules-${ELECTRON_VER}.tar.xz ]; \
+		then ${MKDIR} ${WRKDIR}/npm-cache; \
+		${CP} ${FILESDIR}/package.json \
+			${FILESDIR}/package-lock.json ${WRKDIR}/npm-cache; \
+		cd ${WRKDIR}/npm-cache && \
+		HOME=${WRKDIR} npm ci --verbose --no-progress && \
```

Comment:
```suggestion
		${SETENV} HOME=${WRKDIR} npm ci --verbose --no-progress && \
```

Reply:
I'm not sure this does what's expected? In a bourne-compatible shell, an environment variable is set for just the following command when you write it in front. Does using `setenv` do the same thing, and if so, what's the benefit?

Reply:
The value of `SETENV` is `/usr/bin/env` and it does the same thing according to the man page.

I have followed a (seemingly) FreeBSD ports convention although it seems unnecessary.
(`${SETENV} environment-variables command` is seen here and there in bsd.*.mk and other ports' Makefiles).

This port has `${SETENV}` in `pre-configure` and `do-configure` targets as well.

Reply:
I see, so this is for (bourne) shells that don't support it themselves? Didn't think/know something like that existed. But then, it's pointless until I can find out why you're getting a different tarball....